test/suites/ubuntu-core-edgex-image/model.yaml

```diff
@@ -0,0 +1,43 @@
+type: model
+series: '16'
+# authority-id and brand-id must be set to your developer-id
+authority-id: 3ehWphH4KXEI2BY8mla8ydYPJm8BCfkd
+brand-id: 3ehWphH4KXEI2BY8mla8ydYPJm8BCfkd
+model: ubuntu-core-22-amd64
+architecture: amd64
+# timestamp should be within your signature's validity period
+timestamp: '2023-03-07T13:40:10+00:00'
+base: core22
+grade: dangerous
+snaps:
+  - # This is our custom, dev gadget snap
+    # It has no channel and id, because it isn't in the store.
+    # We're going to build it locally and pass it to the image builder. 
+    name: pc
+    type: gadget
+    # default-channel: 20/stable
+    # id: UqFziVZDHLSyO3TqSWgNBoAdHbLI4dAH
```

Comment:
These aren't correct anymore, since we use 22/stable.

Reply:
I have updated it to `22/stable`, and left it commented.